keyboards/handwired/unk/rev1/config.h

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2020 Herpiko Dwi Aguno <herpiko@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
```

Comment:
You might want to use a unique VID, just in case you or somebody else decides one day that VIA support would be good.

Reply:
@Erovia Interesting! At the moment I want to leave this unchanged. I'll learn about VIA support later.